content/6_Advanced/Vectorization.mdx

```diff
@@ -133,3 +133,265 @@ instead of faster (ex. see
 <!-- Anyways, keep in mind that these rarely make a difference in competitive programming. -->
 
 <!-- <IncompleteSection /> -->
+
+## Example - Ligatures
```

Comment:
this should be a practice problem with an internal solution. see the practice problems with internal solutions at the end of other modules for examples

Reply:
sorry abt that, I have created a second pull request in a new mdx file in the solutions division

Reply:
can you add that to this pull request? should be simple if you're using Github Desktop

Reply:
oky the answer is now in the solutions division in this pull request

Reply:
ok, then can you remove the solution from the module? and make sure the solution is linked from the module (same as internal solution for any other problem)